graphql-apollo-server-tutorial.md

```diff
@@ -2556,7 +2545,9 @@ export const isMessageOwner = async (
 };
 {{< /highlight >}}
 
-This resolver checks whether the authenticated user is a message owner. It's the perfect check before deleting a message if only the message creator should be able to delete it. So the guarding resolver retrieves the message by id, checks the message's associated user with the authenticated user, and either throws an error or continues with the next resolver. Let's protect an actual resolver with this fine-grained authorization permission resolver in the *src/resolvers/message.js* file:
+This resolver checks whether the authenticated user is the message owner. It's a useful check before deleting a message, since you only the message creator to be able to delete it. The guarding resolver retrieves the message by id, checks the message's associated user with the authenticated user, and either throws an error or continues with the next resolver. 
```

Comment:
"since you only the message creator to be able to delete it" -> "since only the message creator is able to delete it"